myhpom/context_processors.py

```diff
@@ -0,0 +1,7 @@
+from django.conf import settings
+
+
+def contact_email(request):
+    return {
+        'contact_email': settings.CONTACT_EMAIL,
```

Comment:
Where does the CONTACT_EMAIL get added to env? Did you add it directly to your .env file? Would it be better to hardcode an email into (one of) the settings.py files instead? If not, should the project README be updated (instructions for config near the top)?

Reply:
I actually have no idea. 😅 

Because it worked _in that one case_, I assumed it was set up properly on staging & that the fact that it is _not_ set up properly on my environment (!) was not a serious problem.

But you're right—this should be documented at the very least.